actions/views/root.js

```diff
@@ -45,40 +56,22 @@ export function loadTranslations(locale, url) {
 
         // No need to go to the server for EN
         if (locale === 'en') {
-            Object.assign(translations, en);
-            dispatch({
-                type: ActionTypes.RECEIVED_TRANSLATIONS,
-                data: {
-                    locale,
-                    translations,
-                },
-            });
+            copyAndDispatchTranslations(dispatch, translations, en, locale);
```

Comment:
This definitely works, however it would've been nice if we had converted this into `async/await` and had just one dispatch in this function rather than dispatching twice as per two different code paths.

```
...
set variables for 'en'
or set variables when fetching through 'Client4'
...

copyAndDispatchTranslations(dispatch, translations, serverTranslations, locale);
```

Reply:
@ali-farooq0, is this something we might yet still be able to squeeze in v5.15? Or is it an internal detail that we can safely improve in v5.16?

Reply:
@lieut-data nah, just an internal detail that we can improve for 5.16. No need to rush it for 5.15.

Reply:
https://mattermost.atlassian.net/browse/MM-18239 to keep track